ui/raidboss/data/04-sb/raid/o7s.js

```diff
@@ -82,11 +82,11 @@
 
     {
       id: 'O7S Magitek Ray',
-      regex: / 14:2788:Guardian starts using (Magitek Ray)/,
-      regexDe: / 14:2788:Wächter starts using (Magitek-Laser)/,
-      regexFr: / 14:2788:Gardien starts using (Rayon Magitek)/,
-      regexJa: / 14:2788:ガーディアン starts using (魔導レーザー)/,
-      regexKo: / 14:2788:가디언 starts using (마도 레이저)/,
```

Comment:
@quisquous whats your opinion on this?
I would replace the alerttext function like in other triggers to have most triggers be consistent without unaccassary functions

Reply:
I am also okay with that change; larger picture, it feels weird for Cactbot to call out the move that's being used as it's being used without auxiliary information since that's provided via in-game features like focus target.

Reply:
The user expirience will be the same with the change for the moment and would need to be adapted per language as usual (i could also just copy the tts text as alert text if prefered)

Reply:
Right, sorry. I should have been a little clearer with my comment. I'm cool with your proposed change, and I know it will all end up being the same, so I say go for it. I think for a lot of these older fights we should work on the hygiene as far as going back and retroactively implementing best practice, but the triggers as they were are probably mostly fine with the information they convey, since there was no urgency to update them back in the day.

I guess my comment was mostly just in general that it feels like an odd trigger to begin with that was returning the spell name as the alert.

Reply:
Ahh ok :) will be added with next commit